keytool/src/main/scala/org/constellation/keytool/Base58.scala

```diff
@@ -51,4 +51,14 @@ object Base58 {
     )
     if (trim.isEmpty) zeroes else zeroes ++ decoded.toByteArray.dropWhile(_ == 0) // BigInteger.toByteArray may add a leading 0x00
   }
+
+  /**
+    * Checks if the input contains base58 chars only
+    *
+    * @param input string to check
+    * @return true if input contains base58 chars only, false otherwise
+    */
+  def isBase58(input: String): Boolean = {
+    input.forall(alphabet.toArray.contains)
```

Comment:
It's a minor thing, so it's up to you if you want to fix that, but if you would replace `.toArray` with `.toSet` you would have O(n) instead of O(n^2)